templates/miq-template.yaml

```diff
@@ -691,10 +771,14 @@ parameters:
   displayName: Memcached Slab Page Size
   description: Memcached size of each slab page.
   value: 1m
+- name: POSTGRESQL_CONFIG_DIR
+  displayName: PostgreSQL Configuration Overrides
+  description: Directory used to store PostgreSQL configuration overrides.
+  value: "/var/lib/pgsql/conf.d"
 - name: POSTGRESQL_MAX_CONNECTIONS
   displayName: PostgreSQL Max Connections
   description: PostgreSQL maximum number of database connections allowed.
-  value: '100'
+  value: '1000'
 - name: POSTGRESQL_SHARED_BUFFERS
```

Comment:
@carbonin 😄 Sure we can go ahead with the 1GB SHARED_BUFFERS default change for this one, I'll commit in a moment.